src/main/java/windycall/Ui.java

```diff
@@ -0,0 +1,81 @@
+package windycall;
+
+import java.util.List;
+
+/**
+ * Deals with interactions with the user
+ * Makes the background of chatBox more pretty
+ */
+public class Ui {
+
+    public Ui() {
+
+    }
+
+    public static void space() {
+        System.out.print("     ");
+    }
```

Comment:
I notice a lot of ui.space() followed by System.out.println().
Perhaps consider abstracting further and have a ui.println() ?
It would also make it more clear that space is writing something to the user.